### PR TITLE
feat(github): add create-milestone action

### PR DESCRIPTION
**Pull request by xzf0587**
https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/21987041

### CODE REVIEW

**Comment by xzf0587** [comment]
run action: https://github.com/OfficeDev/TeamsFx/actions/runs/5195920654

File: .github/workflows/create-milestone.yml [hunk 1]
```diff
@@ -9,18 +9,25 @@ jobs:
...
+        run: cp -r .github/actions/create-milestone ./action-base/create-milestone
+      - name: Npm install
+        run: npm install --production --prefix ./action-base
+      - name: Build action
+        continue-on-error: true
+        run: npm run build --prefix ./action-base
 
       - name: Run Create Milestone
-        uses: ./actions/create-milestone
+        uses: ./action-base/create-milestone
```
Comment (wenytang-ms) [inline .github/workflows/create-milestone.yml:30]: the path exist?

Comment (wenytang-ms) [reply .github/workflows/create-milestone.yml:30]: but I don't find the create-milestone folder in the repo?

**Review by wenytang-ms — APPROVED** [review]

File: .github/workflows/create-milestone.yml [hunk 2]
```diff
@@ -9,18 +9,25 @@ jobs:
...
-         path: ./actions
-         token: ${{secrets.GHEC_BOT_PAT}}
-      - name: Install Actions
-        run: npm install --production --prefix ./actions
+         repository: "microsoft/vscode-github-triage-actions"
+         ref: stable 
+         path: ./action-base
+
+      - name: Copy action
+        run: cp -r .github/actions/create-milestone ./action-base/create-milestone
```
Comment (wenytang-ms) [inline .github/workflows/create-milestone.yml:22]: before copy the repo's file, should this action checkout this TeamsFx first?

**Review by wenytang-ms — CHANGES_REQUESTED** [review]
should checkout current repo before cp action

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**